submissions/Oleksii_Drohachov/HTML_CSS_PopUp/index.html

```diff
@@ -0,0 +1,228 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <link rel="stylesheet" href="fonts/Ubuntu-Medium.ttf" />
+    <title>Hooli-style-PopUp</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a
+              class="nav_link"
+              href="https://github.com/kottans/frontend/blob/2022_UA/contents.md"
+              >My lovely course</a
+            >
+          </li>
+          <li class="nav__item">
+            <a class="nav_link" href="https://t.me/+kI9jsoLH2rQ4YzVi"
+              >My Kottans in TG</a
+            >
+          </li>
+          <li class="nav__item__menu">
+            <label for="first"></label>
+            <input id="first" class="first" type="checkbox" name="first" />
+            <div class="main__wrapper">
```

Comment:
Why you don't use `ul` inside `li` Logically it looks like the second level of navigation.